code/src/Models/User.ts

```diff
@@ -82,13 +119,21 @@ export default class User{
         }
         
     }
-    
-    
-
 }
 
 function parseError(error:string){
-    //example error auth/email-already-exists to Email Already Exists
-    // Could be used to communicate with user in front end
-    return error.split('/')[1].split('-').map((e)=>(e.charAt(0).toUpperCase()+e.slice(1)))
+    /**
+     * @param error - firebase error code from  [https://firebase.google.com/docs/auth/admin/errors]
+     * converts error to user friendly string
+     * @return user friendly string
+     */
+    switch (error){
```

Comment:
Do you think it would be useful to create an Errors class? That way we can classify and handle errors around the code better.

Reply:
Hmm yeah that sounds more maintainable - will do

Reply:
Probs should be a return from login() and signUp()
Promise<User|FireError>